modules/sqlauth/lib/Auth/Source/SQL.php

```diff
@@ -68,6 +74,7 @@ public function __construct($info, $config) {
 		$this->dsn = $config['dsn'];
 		$this->username = $config['username'];
 		$this->password = $config['password'];
+		$this->options = $config['options'];
```

Comment:
Here options is used unconditionally thus mandatory part of the config, while above it's optional. This should be consistent (and for BC I'd want it to be optional).